src/request/TopLevelApi.js

```diff
@@ -229,15 +229,60 @@ class TopLevelApi extends RequestParser {
     }
 
     /**
-     * @param {string} buttonText
+     * @param {string|Parsed<T>} requestOrCustomButtonText
      */
-    setGlobalCloseButtonText(buttonText) {
+    enableGlobalCloseButton(requestOrCustomButtonText) {
         /** @type {HTMLElement} */
         const $globalCloseText = (document.querySelector('#global-close-text'));
         /** @type {HTMLSpanElement} */
         const $button = ($globalCloseText.parentNode);
         if (!$button.classList.contains('display-none')) return;
-        $globalCloseText.textContent = buttonText;
+
+        // eslint-disable-next-line require-jsdoc-except/require-jsdoc
+        const setButtonText = () => {
```

Comment:
```suggestion
        function setButtonText() {
```

Reply:
In my opinion the way it is currently is the nicer way of defining a function within a function.